### ISSUE TITLE
Cannot query the value of task ':mkdocsBuild' property 'extraArgs' because it has no value available

### ISSUE THREAD

**Issue by JavierSegoviaCordoba** [xvik/gradle-mkdocs-plugin#88]
After upgrading to 4.0.0, I am getting this error:

```
  * What went wrong:
  Execution failed for task ':mkdocsBuild'.
  > Cannot query the value of task ':mkdocsBuild' property 'extraArgs' because it has no value available.
```

**Comment by xvik** [comment]
Sorry, my non-strict tests (with `mkdocs.strict=false`) was not quite correct so I missed this silly mistake.
Thank you very much for reporting!

I'll release fixed plugin today

**Comment by xvik** [comment]
[released 4.0.1](https://github.com/xvik/gradle-mkdocs-plugin/releases/tag/4.0.1)
